### PR TITLE
PXB-2467 (Create helper function for Group Replication)

### PR DESCRIPTION
**Pull request by altmannmarcelo**
https://jira.percona.com/browse/PXB-2467

Problem:
Currently we don't test GR as part of PXB testing.

Solution:
Implemented helper functions to bootstrap a GR cluster into single write
mode.
Enhanced the test suite to understand the concept of suites.
Current tests under t/ folder will be called main suite.
Added gr suite with a simple test case in it.
By default, run.sh will invoke main and gr suite. This can later be
extended for PXC.

### CODE REVIEW

**Comment by altmannmarcelo** [comment]
Output of test framework changed a bit:
```
./run.sh
Using /work/pxb/ins/8.0/xtrabackup-test/test/var as test root
Detecting server version...
Running against Percona Server 8.0.22-13-debug (XtraDB 8.0.22-13)
Using 'xtrabackup' as xtrabackup binary
Autodetected number of cores: 32
Limiting to 16 to avoid excessive resource consumption
Using 16 parallel workers
Running suite(s): main gr

==============================================================================
TEST                                   WORKER    RESULT     TIME(s) or COMMENT
------------------------------------------------------------------------------
main.argument_handling                   w1	[passed]    1
gr.gr_basic                              w2	[passed]    54
==============================================================================
Spent 55 of 68 seconds executing testcases

SUMMARY: 2 run, 2 successful, 0 skipped, 0 failed

See results/ for detailed output
```

We now display the running suites (when more than one):
```
Running suite(s): main gr
```
We also prepend the suite on pass/fail display:
```
main.argument_handling                   w1	[passed]    1
gr.gr_basic                              w2	[passed]    54
```

**Comment by altmannmarcelo** [comment]
https://pxb.cd.percona.com/job/percona-xtrabackup-8.0-test-param/25/

File: storage/innobase/xtrabackup/test/inc/common.sh [hunk 1]
```diff
@@ -445,6 +486,92 @@ function shutdown_server()
...
+gtid_mode=ON
+enforce_gtid_consistency=ON
+plugin_load_add='group_replication.so'
+loose-group_replication_group_name=\"aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa\"
+loose-group_replication_start_on_boot=off
+loose-group_replication_bootstrap_group= off
+loose-group_replication_recovery_use_ssl=ON
+loose-group_replication_recovery_get_public_key=ON
+loose-group_replication_local_address=\"127.0.0.1:${SRV_MYSQLD_GR_PORT[$i]}\"
+loose-group_replication_group_seeds= \"${GR_CLUSTER_ADDRESS: : -1}\""
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/test/inc/common.sh]: could you use loose-group_replication_group_seeds= \"${GR_CLUSTER_ADDRESS%?}\""
:: is not working on Darwin.

**Review by rahulmalik87 — APPROVED** [review]
LGTM

**Pull request merged**